capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -0,0 +1,95 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.time.LocalDateTime;
+import java.util.*;
+
+@Entity(name = "event")
+public class Event{
+
+  @Id
+  Long datastoreID;
+
+  @Field(name="organizationName")
+  String organizationName;
+
+  @Field(name="eventTitle")
+  String eventTitle;
+
+  @Field(name="eventDateTime")
+  String eventDateTime;
+
+  @Field(name="eventDescription")
+  String eventDescription;
+
+  // @Field(name="university")
+  // String university;
+
+  @Field(name="eventLatitude")
+  double eventLatitude;
+
+  @Field(name="eventLongitude")
+  double eventLongitude;
+
+  // @Field(name="eventFilters")
+  // List<String> eventFilters;
```

Comment:
Would those be filters or "categories"?

Reply:
Categories is probably a better description, but I used 'filters' for continuity's sake